packages/uniforms-material/src/ListDelField.tsx

```diff
@@ -30,19 +30,16 @@ function ListDel({
     { absoluteName: true },
   )[0];
 
-  const limitNotReached =
-    !disabled && !(parent.minCount! >= parent.value!.length);
+  disabled ||= readOnly || parent.minCount! >= parent.value!.length;
```

Comment:
I know it's not an object, but is it a good practice to override the `disabled` value?